Extensions/Cosmos/README.md

```diff
@@ -7,10 +7,12 @@ The Cosmos data transfer extension provides source and sink capabilities for rea
 ## Settings
 
 Source and sink require settings used to locate and access the Cosmos DB account. This can be done in one of two ways:
+
 - Using a `ConnectionString` that includes an AccountEndpoint and AccountKey
-- Using RBAC (Role Based Access Control) by setting `UseRbacAuth` to true and specifying `AccountEndpoint` and optionally `EnableInteractiveCredentials` to prompt the user to log in to Azure if default credentials are not available.
+- Using RBAC (Role Based Access Control) by setting `UseRbacAuth` to true and specifying `AccountEndpoint` and optionally `EnableInteractiveCredentials` to prompt the user to log in to Azure if default credentials are not available. See ([migrate-passwordless](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/migrate-passwordless?tabs=sign-in-azure-cli%2Cdotnet%2Cazure-portal-create%2Cazure-portal-associate%2Capp-service-identity) for how to configure Cosmos DB for passwordless access.
```

Comment:
Can you remove the language-locale in this URL? If there is a translated version of this doc MS Learn will direct the user to it vs. sending to US English only.

Reply:
Good catch. I had a tough time finding any resource that indicated how to link to Microsoft Learn documentation. I did find a mention of the don't include locale in [Use Links in documentation](https://learn.microsoft.com/contribute/content/how-to-write-links#links-to-pages-that-are-not-part-of-the-technical-documentation-set). I'll remove the en-us.

Reply:
en-us removed.